icd/api/compiler_solution_llpc.cpp

```diff
@@ -354,6 +354,34 @@ VkResult CompilerSolutionLlpc::CreateGraphicsPipelineBinary(
     {
         *ppPipelineBinary   = pipelineOut.pipelineBin.pCode;
         *pPipelineBinarySize = pipelineOut.pipelineBin.codeSize;
+        if (pipelineOut.pipelineCacheAccess !=
+            Llpc::CacheAccessInfo::CacheNotChecked)
```

Comment:
Can this fit on one line?  The line limit is 120 characters.

Reply:
done